### PR TITLE
Flesh out READMEs and other docs

### PR DESCRIPTION
**Pull request by grassdog**

### CODE REVIEW

File: app/aggregates/todo.rb [hunk 1]
```diff
@@ -3,8 +3,15 @@ module Aggregates
...
+      # These apply methods are the hook that this aggregate uses to update
+      # its internal state from events.
+
       apply TodoAdded do |event|
-        @added = true
+        # We track our ID when we are added to we can ensure the same todo isn't
+        # added twice.
+        #
+        # We could save more attributes off the event in here if we needed them.
+        @aggregate_id = event.aggregate_id
```
Comment (grassdog) [inline app/aggregates/todo.rb]: Wanted to demonstrate actually storing state from the event at least once.

File: Rakefile [hunk 2]
```diff
@@ -12,10 +18,11 @@ task run_processors: :environment do
   tracker = EventSourceryTodoApp.tracker
   db_connection = EventSourceryTodoApp.projections_database
 
-  # Need to disconnect before starting the processors
+  # Need to disconnect before starting the processors so
+  # that the forked processes have their own connection / fork safety.
```
Comment (twe4ked) [inline Rakefile]: :100:

File: app/projections/README.md [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# Projections
+
+You can think of projections as read models. They are created and updated by projectors and in this case show different current state views over the events that are the source of truth for our application state.
```
Comment (twe4ked) [inline app/projections/README.md]: read-only models?

File: app/aggregates/todo.rb [hunk 4]
```diff
@@ -3,8 +3,15 @@ module Aggregates
     class Todo
       include EventSourcery::AggregateRoot
 
+      # These apply methods are the hook that this aggregate uses to update
+      # its internal state from events.
+
       apply TodoAdded do |event|
-        @added = true
+        # We track our ID when we are added to we can ensure the same todo isn't
```
Comment (pablolee) [inline app/aggregates/todo.rb]: `We track our ID when we are added to we can/We track the ID when the event todo is added to`

**Pull request merged**